bundler/doc/contributing/HOW_YOU_CAN_HELP.md

```diff
@@ -6,22 +6,22 @@ If at any point you get stuck, here's how to [get in touch with the Bundler team
 
 ## First contribution suggestions
 
-We track [small bugs and features](https://github.com/rubygems/bundler/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) so that anyone who wants to help can start with something that's not too overwhelming.
+We track [small bugs and features](https://github.com/rubygems/rubygems/issues?q=is%3Aissue+is%3Aopen+label%3Abundler++label%3A%22good+first+issue%22+) so that anyone who wants to help can start with something that's not too overwhelming.
```

Comment:
Sorry to bother you again, I just noticed a couple of extra "+" signs here (they seem to be handled just fine, but still).

```suggestion
We track [small bugs and features](https://github.com/rubygems/rubygems/issues?q=is%3Aissue+is%3Aopen+label%3Abundler+label%3A%22good+first+issue%22) so that anyone who wants to help can start with something that's not too overwhelming.
```

Reply:
Nice, thanks. Now if you can do the squashing again, it's ready to go!